packages/protocol/contracts/common/GoldToken.sol

```diff
@@ -256,11 +294,11 @@ contract GoldToken is
 
   /**
    * @notice Gets the balance of the specified address.
-   * @param owner The address to query the balance of.
+   * @param _owner The address to query the balance of.
```

Comment:
the _ has been added because the variable `owner` is already defined as a state variable of the contract. This was more of a NIT to avoid shadowed variables/unexpected behaviors.